compiler/src/typed/module_resolution.re

```diff
@@ -75,14 +75,18 @@ let find_ext_in_dir = (dir, name) => {
   process_ext;
 };
 
-let find_in_path_uncap = (~exts=[], path, name) => {
+let is_relpath = path =>
+  Filename.is_relative(path) && !Filename.is_implicit(path);
```

Comment:
Does `Filename.is_implicit` make assumptions about the ocaml compiler?

Reply:
@phated I'm not sure how you mean...can you elaborate? This is the documentation for the function:

> Return true if the file name is relative and does not start with an explicit reference to the current directory (./ or ../ in Unix), false if it starts with an explicit reference to the root directory or the current directory.

Reply:
Sorry, I didn't look up the docs and assumed it had something to do with implicit modules that ocaml knows about. Don't mind me 🙈